lib/tags.ts

```diff
@@ -17,15 +17,22 @@ export function getTags() {
 export function getTagSlugs() {
   const tags = getTags()
 
-  return map((tag: any = {}) => {
-    const {slug, label} = tag
-    return {
-      params: {
-        slug,
-        label,
-      },
-    }
-  })(tags)
+  type Tag = {
+    slug: string
+    label: string
+  }
+  return pipe(
+    filter((tag: Tag) => tag.slug != 'react'),
```

Comment:
@joelhooks This fixed the build. The tags.json was generating a path for `learn/react` that was interfering with the mdx file. If we stick to using `tags.json` to generate paths, we could make this filter smarter by checking the `learn` path for existing folders. I'm leaving as a hard-coded `"react"` for now.

Reply:
Good catch! Thanks.